src/borg/testsuite/archiver/debug_cmds.py

```diff
@@ -63,6 +63,55 @@ def test_debug_put_get_delete_obj(self):
         output = self.cmd(f"--repo={self.repository_location}", "debug", "delete-obj", "invalid")
         assert "is invalid" in output
 
+    def test_debug_id_hash_format_put_parse_obj(self):
+        self.cmd(f"--repo={self.repository_location}", "rcreate", RK_ENCRYPTION)
+        data = b"some data"
+        meta = b'{"some" : "property"}'
```

Comment:
guess this is a bit backwards. :-)

instead of "hand writing" json, start with defining the meta_dict and then json.dumps that into meta_json.

that will work easier and makes sure the json is valid, even if the metadata would be a bit more complicated.